pkg/kapp/app/interfaces.go

```diff
@@ -19,9 +19,8 @@ type App interface {
 
 	LabelSelector() (labels.Selector, error)
 	UsedGVs() ([]schema.GroupVersion, error)
-	UpdateUsedGVs([]schema.GroupVersion) error
 	UsedGKs() (*[]schema.GroupKind, error)
```

Comment:
nit: Do we need a reference here. I am asking because `UsedGVs()` doesn't return the reference.

Reply:
Yep we use it to determine whether an existing app was created by the older or newer kapp.
 This pointer would be nil for older apps as we set it [here](https://github.com/vmware-tanzu/carvel-kapp/commit/98b377c63a797aa00c50867ea7c2129cabb0c709#diff-c4cadf071a66c0e934e58c35eb0a9116fee4c744abd14621dc1eeebe7f5e9fb8R133)